prometheus_django/middleware.py

```diff
@@ -2,6 +2,15 @@
 import time
 from prometheus_client import Counter, Summary, Gauge
 
+try:
+    # MiddlewareMixin is for backwards compatibility from django >=1.10
+    # to earlier versions, so it didn't exist before django 1.10.
+    # In our older Microservices we don't need to use this.
+    # https://docs.djangoproject.com/en/2.2/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware
+    from django.utils.deprecation import MiddlewareMixin
+except ImportError:
+    MiddlewareMixin = object
+
```

Comment:
you need to inherit from `MiddlewareMixin` in your middleware class

Reply:
oh yeah, good spot. Amended commit with fix.